versioned_docs/version-0.4.2/process-and-procedures/naming-convention.md

```diff
@@ -110,7 +110,7 @@
 
 ## yEarn
 
-- These are Yearn's original yield-aware tokens, whose v2 and v3 contracts can be found [here](https://docs.yearn.finance/getting-started/products/yvaults/vault-tokens#v2-yvault-tokens).
```

Comment:
The two links are very different stuff. This can show where to find the token addresses in the UI so people can check on etherscan. There's a link to the contract in the new UI